src/tech/v3/dataset/print.clj

```diff
@@ -216,3 +216,31 @@ tech.ml.dataset.github-test> (def ds (with-meta ds
              (dataset-data->str ds options))))
   ([ds]
    (dataset->str ds {})))
+
+
+(defn print-range
+  "Convenience function to set the number of rows to print."
```

Comment:
Perhaps some documentation stating that it takes a range (and perhaps :all as we discussed).